src/GrpcService.ts

```diff
@@ -2,7 +2,7 @@ import * as semver from 'semver';
 
 if (!semver.satisfies(process.version, '8.x')) {
   let errorMessage = `Your Function App is currently set to use current version ${process.version}, but the runtime requires Node 8.x.
-  For delpoyed code, please change WEBSITE_NODE_DEFAULT_VERSION in App Settings. On your local machine, you can change node version using 'nvm'`;
+  For delpoyed code, please change WEBSITE_NODE_DEFAULT_VERSION in App Settings. On your local machine, you can change node version using 'nvm' (make sure to 'Quit' VSCode and start again instead of reloading it when you change the node version)`;

```

Comment:
Typo here should say `deployed` instead of `delpoyed`

Reply:
Oh, it was there before. But I'll go ahead and fix that.